ignite/services/plugin/plugin.go

```diff
@@ -202,7 +202,7 @@ func (p Plugin) Manifest() *Manifest {
 }
 
 func (p Plugin) binaryName() string {
-	return fmt.Sprintf("%s.app", p.name)
+	return fmt.Sprintf("%s.ign", p.name)
```

Comment:
Isn't that breaking for v28? Shouldn't there be some sort of depreciation path?

Reply:
It should be fine because the app binaries will be rebuilt automatically when users run an app command for the first time. The "downside" is that right now the ".app" binaries won't be removed but that should not be an issue.

In any case this PR is just a proposed fix. The only breaking issue right now happens in macOS Sonoma, which I think is the last released macOS version.